### ISSUE TITLE
[A11yInspect] Fail: 4.1.2 A in /home/runner/work/test-app/test-app/src/sample.jsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#9]
## ❌ Accessibility Fail

**Rule:** 4.1.2 A
**File:** `/home/runner/work/test-app/test-app/src/sample.jsx`
**Line:** 83, **Column:** 1

### Description
Missing accessible name for combobox (Major)

### Context
```html
<select>
            <option>Select country</option>
          </select>
```

### Selector
`html > body > div > div > video > audio > form > select`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
